plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -401,7 +401,7 @@ instance (KnownBuiltinTypeIn DefaultUni term Integer, Integral a, Bounded a, Typ
             -- TODO: benchmark an alternative 'integerToIntMaybe', modified from @ghc-bignum@
             if fromIntegral (minBound :: a) <= i && i <= fromIntegral (maxBound :: a)
                 then pure . AsInteger $ fromIntegral i
-                else throwing _OperationalUnliftingError . MkUnliftingError $ fold
+                else throwOperationalUnliftingError . MkUnliftingError $ fold
```

Comment:
```suggestion
                else throwError . mkUnliftingError $ fold
```
(Don't apply, this is an example of idea)

Reply:
Ah, yes `UnliftingError ~ Text` anyways, and on other cases we were just relying on `IsString UnliftingError` anyways. Making `operationalUnliftingError :: Text -> BuiltinError` would be most reasonable; I agree.